.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
+
+env:
+  RACK_ENV: development
+  S3_REAL: true
+  S3_REGION: us-east-1
+  S3_BUCKET: minio-bucket
+  S3_ACCESS_KEY_ID: weak_key
+  S3_SECRET_ACCESS_KEY: weak_key
+  S3_ENDPOINT: http://localhost:9000
+  MINIO_ACCESS_KEY: weak_key
+  MINIO_SECRET_KEY: weak_key
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - 2.3
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+          # - ruby-head
+          - jruby-9.2
+          # - jruby-head
+
+    name: testing under ruby ${{ matrix.ruby }}
```

Comment:
I haven't yet figured out what I think is the best naming convention here to lead to good output in all the various places GH Actions shows it. This can be changed to whatever one wants.

Reply:
I'd like having just `${{matrix.ruby}}`